root/static/scripts/guess-case/utils.js

```diff
@@ -144,8 +152,8 @@ exports.titleString = function (is, forceCaps) {
     }
 
     var os;
-    var lc = is.toLowerCase();
```

Comment:
I think what @uklauer meant is that only the changes from here down, plus the `toLowerCase`/`toUpperCase` methods added to `DefaultMode`, should be in the first commit. The message can be something like "Add support for mode-specific casing functions".

Everything else that has to do with the Turkish mode specifically should be in the second commit.